cli/policy.go

```diff
@@ -365,9 +339,12 @@ func importPolicy(ctx *cli.Context) {
 			return
 		}
 
-		if err := client.PolicyAdd(node.Name, node); err != nil {
+		jsonPolicy := node.JSONMarshal()
+		if resp, err := client.PolicyPut(node.Name, jsonPolicy); err != nil {
 			fmt.Fprintf(os.Stderr, "Could not import policy directory %s: %s\n", path, err)
 			os.Exit(1)
+		} else if ctx.Bool("echo") {
```

Comment:
When we are importing a policy with the CLI we don't have any feedback what happen with the operation. What about `fmt.Fprintf(os.Stderr, "Policy...")`?